packages/checkout-ui-extensions/src/components/Divider/content/guidelines.md

```diff
@@ -0,0 +1,9 @@
+## Best practices
+
+- When using a divider with text, keep the text short and succinct so that it doesn’t wrap into two lines, or cause the divider line to disappear
+- Don’t stack dividers or place them too close together
+- Use the default divider width unless you have a strong reason not to
+
+## Related components
+
+- [InlineSpacer](https://github.com/Shopify/checkout-web/tree/master/packages/checkout-ui-extensions/src/components/InlineSpacer): Controls vertical margin.
```

Comment:
```suggestion
- [InlineSpacer](https://github.com/Shopify/checkout-web/tree/master/packages/checkout-ui-extensions/src/components/InlineSpacer): Controls inline margin.
```

Reply:
Is this supposed to be BlockSpacer?

Reply:
You're right, this is actually BlockSpacer

Reply:
Could put both components because the divider can be used horizontally and vertically (inline and block)